dhcpcd-hooks/65-kano-dhcp-hook

```diff
@@ -38,24 +36,60 @@ REBOOT | BOUND)
     # Try uploading the tracking data to our servers
     # Should be quiet on failure
     kano-sync --upload-tracking-data --silent &
+}
 
-    # Sync objects from the content API in the background
-    # TODO: This should be removed once we have the daemon done
+# Sync objects from the content API in the background
+# TODO: This should be removed once we have the daemon done
+kano_daemon () {
     kano-content sync &
-
     kano-tracker-ctl +1 'internet-connection-established' &
-    ;;
+}
 
-NOCARRIER | STOPPED)
-    # Are we still networked due to multihomed connection?
+# Check for a multihomed connection and keep it if there
+# otherwise disconnect with a logger notification
+is_multihomed () {
     is_internet
     if [ $? -eq 0 ]; then
-    	logger "kano-dhcp-hook: NOCARRIER/STOPPED but still connected, aborting action"
+    	logger "kano-dhcp-hook: $reason but still connected, aborting action."
     	exit 0
```

Comment:
Same as above for `return` ?

Reply:
Thanks, @skarbat. 

Indeed, I can return a value (e..g 0 for fail) from both functions and then test it inside the script to exit. Do you think this would be a more suitable generalised approach? If so I am going to change.

Reply:
Oh, true. Well, given that there is a clear log on exit I think it's all good as it is.
👍 @wizofe